cron/poll_read_issue_comments.py

```diff
@@ -246,26 +237,51 @@ def handle_comment(api, issue_comment):
 
         # We doin stuff boyz
         if can_run and has_votes:
-            __log.debug("Handling issue {issue}: comment {comment}".format(issue=issue_id,
+            __log.debug("Handling issue {issue}: command {comment}".format(issue=issue_id,
                                                                            comment=comment_text))
 
             if command == "/vote":
                 handle_vote_command(api, parsed_comment, issue_id, global_comment_id, votes)
+
+            update_command_ran(api, global_comment_id, "Command Ran")
+
         elif can_run and not has_votes:
             # oops we didn't pass
-            db = update_db(global_comment_id, {"has_ran": True})
-            db_fields = select_db(global_comment_id, ("chaos_response_id", "command"), db=db)
-            resp_id = db_fields["chaos_response_id"]
-            command = db_fields["command"]
-            body = "> {command}\n\nVote failed".format(command=command)
-            gh.comments.edit_comment(api, settings.URN, resp_id, body)
+            update_command_ran(api, global_comment_id, "Vote Failed")
 
 
 def is_command(comment):
```

Comment:
I'll look into getting some. The next step is to refactor the code to use mysql. Should make everything cleaner and easier to debug